src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './NavBar.css';
+
+const NavBar = () => (
+  <div className="containerNavBar">
+    <div className="NavBar">
+      <div>
+        <h1>Bookstore CMS</h1>
+      </div>
+      <div className="links">
+        <Link to="/Books" className="books">Books</Link>
```

Comment:
- [ ] Kindly use the root path for your routing here to keep it uniform with the app route.

  ```
  <Link to="/" className="books">Books</Link>
  ```